### DISCUSSION TITLE
How to enable GitHub Mobile as a 2FA app?

### DISCUSSION THREAD

**Discussion by mybearworld** [community/community#106528] [Mobile] [answered]
### Select Topic Area

Question

### Body

I can't find an option to do so anywhere.

**Comment by mybearworld** [comment]
Funnily enough, [most people seem to be having the opposite problem...](https://github.com/orgs/community/discussions/31942)

**Comment by KPROsoft21** [comment]
yep

**Comment by gil-son** [comment]
Hello,

1. Click on your profile picture and click on Settings:
![g01](https://github.com/community/community/assets/72712095/6c0b89a8-c879-4f20-b082-35399fea08e7)

2. Click on Password and authentication:
![g02](https://github.com/community/community/assets/72712095/d2b7e768-6a04-43d7-b14b-02904bbcc4a0)

3. Now you can manage to 2AF, just click on Authenticator App:
![g03](https://github.com/community/community/assets/72712095/e57f3b77-4b45-4c7a-b3c2-fff322509715)

I recommend you to use the Google Authenticator App:

https://www.youtube.com/watch?v=zzY32wTAxKU


If I supported you, please consider marking this answer. God bless you!

**Comment by DNin01** [comment, marked as answer]
2FA via GitHub Mobile isn't actually something you set up - once you activate 2FA with another method such as an authenticator app, all your devices signed into GitHub Mobile will automatically be able to receive authentication requests.

Once you've got it working, you can also set it as your preferred method, so it is used by default.

Note that I've experienced that sometimes GitHub Mobile sessions don't show up in the settings. Opening GitHub Mobile on a device where you're logged in might make it appear again.

**Reply by mybearworld** [reply]
Thank you, I didn't know that. It works now!

**Comment by elhizazi1** [comment]
Response

**Comment by fdzi1** [comment]
[_gvkwt[gi]()**_]()

**Comment by fdzi1** [comment]
- @mybearworld
